content/en/service_management/events/_index.md

```diff
@@ -16,60 +16,34 @@ further_reading:
 
-Without any additional setup, Datadog Events automatically gathers events that are collected by the Agent and installed integrations.
+Ingest, enrich & normalize, and correlate (see public beta) your events from any source into actionable insights. Datadog automatically creates events from various products including monitors, Watchdog, and Error Tracking. You can also track events generated from Agent and installed integrations. Event Management can also ingest events from any sources, including alert events from 3rd parties, change requests, deployments, configuration changes.
 
 More than 100 Datadog integrations support events collection, including [Kubernetes][1], [Docker][2], [Jenkins][3], [Chef][4], [Puppet][5], [Amazon ECS][6] or [Autoscaling][7], [Sentry][8], and [Nagios][9]. 
 
-## Sending custom events to Datadog
-
-You can also submit your own custom events using the [Datadog API][10], [Custom Agent Check][11], [DogStatsD][12], or the [Events Email API][13].
-
-## Exploring Datadog Events
-
-### Events explorer and analytics
-
-Use the [Events Explorer][14] to aggregate and view events coming into Datadog. Group or filter events by attribute and graphically represent them with [event analytics][15]. Use the [query syntax][16] to filter events using Boolean and wildcard operators.
-
-{{< img src="service_management/events/events-explorer.mp4" alt="Sorting events by attributes and exploring analytics" video=true >}}
-
-### Events as a source in dashboards widgets
-
-You can use events as a data source in [graph widgets][17]. You can build timeseries, tables, and top list widgets of your event search queries.
-
-{{< img src="service_management/events/events-dashboard.mp4" alt="A graph widget that uses events as the source" video=true >}}
-
-For example, check out the [Monitor Notifications Overview][18] dashboard, which analyzes monitor alert event trends to help you improve your configuration and reduce alert fatigue.
-
-### Generate custom metrics from events 
-
-[Generate metrics][15] with 15-month retention from any event search query to create and monitor historical events and alerts.
-
-{{< img src="service_management/events/generate-metrics.png" alt="Image of metrics with the events search query." >}}
-
-### Normalize and enrich events with processing pipelines
-
-A _processor_ runs data-structuring actions on events attributes when they are ingested. A _pipeline_ is composed of one or multiple processors executed sequentially. With event processing pipelines, you can:
-
-- Normalize disparate sources of events by remapping attributes. For example, use the same reserved [service tags][19] everywhere.
-- Enrich events with external data saved in a [Reference Table][20] (beta). For example, map a service name with your service directory to enrich events with team ownership information, links to dashboards, or links to documentation.
+{{< img src="service_management/events/events_explorer.png" alt="Events explorer display" >}}
```

Comment:
I noticed that this screenshot has a Datadog employee email in it. I would recommend retaking this screenshot. If you need to, you can change the email displayed in the UI (and make it something generic) using developer tools > Inspect in the browser. Reach out if you need any help doing that.

Reply:
I reevaluated whether we need another picture of the Events Explorer and removed. Thanks for catching this!